path/control_flow.go

```diff
@@ -6,17 +6,17 @@ package path
 
 import (
 	"github.com/gonum/graph"
-	"github.com/gonum/graph/internal"
+	"github.com/gonum/graph/internal/set"
 )
 
 // PostDominatores returns all dominators for all nodes in g. It does not
 // prune for strict post-dominators, immediate dominators etc.
 //
 // A dominates B if and only if the only path through B travels through A.
-func Dominators(start graph.Node, g graph.Graph) map[int]internal.Set {
-	allNodes := make(internal.Set)
+func Dominators(start graph.Node, g graph.Graph) map[int]set.Nodes {
```

Comment:
This and below touches an outstanding issue, that clients don't have access to the documentation of set.Nodes because it is internal. They can get the information, out of the value though.

Just commenting here to note that I am aware of the issue, but it's not a high priority since no-one has complained.